RoktDemo/UI/Layout Demo/LayoutDemoViewModel.swift

```diff
@@ -52,9 +52,35 @@ class LayoutDemoViewModel: ObservableObject {
         guard let preview else { return [:] }
         var attributes = [String: String]()
         attributes["isDemo"] = "true"
-        attributes["layoutId"] = preview.previewId
-        attributes["versionId"] = preview.versionId
-        attributes["creativeId"] = preview.creativeIds.joined(separator: ",")
+        attributes["rokt.language"] = preview.language
+        
+        var slots: [[String: String]] = []
+        let layoutVariantCount = preview.layoutVariantIds.count
+        
+        for (index, creativeId) in preview.creativeIds.enumerated() {
+            let layoutVariantId = preview.layoutVariantIds[index % layoutVariantCount]
+            let slot: [String: String] = [
+                "layoutVariantId": layoutVariantId,
+                "creativeId": creativeId
+            ]
+            slots.append(slot)
+        }
```

Comment:
Updated the logic to accept any number of layoutVariantIds (at least 1), loop through creativeIds, use layoutVariantIds one by one